.gitignore

```diff
@@ -1,3 +1,6 @@
 node_modules
+<<<<<<< HEAD
```

Comment:
@Patrickcpoole Please remove any merge conflict comments, otherwise app will crash

Reply:
& when this is updated, all the build/... files will disappear from this PR & clean it up a lot.

Reply:
Conflicts resolved.